mobile/src/store/courses/actions.ts

```diff
@@ -47,4 +49,26 @@ const addCourse = createAsyncThunk<
   return course;
 });
 
-export { addCourse, getCourse, getCourses };
+const getCategories = createAsyncThunk<
+  CategoryGetAllResponseDto,
+  void,
+  AsyncThunkConfig
+>(ActionType.GET_CATEGORIES, async (_, { extra }) => {
+  const { categoriesApi } = extra;
+  const categoriesDto = await categoriesApi.getAll();
+
+  return categoriesDto;
+});
```

Comment:
Think it's unnecessary, in case that we almost have configured store for categories